shell/platform/windows/flutter_windows_view.cc

```diff
@@ -49,10 +52,39 @@ void FlutterWindowsView::SetEngine(
                     binding_handler_->GetDpiScale());
 }
 
-void FlutterWindowsView::OnWindowSizeChanged(size_t width,
-                                             size_t height) const {
-  surface_manager_->ResizeSurface(GetRenderTarget(), width, height);
+uint32_t FlutterWindowsView::GetFrameBufferId(size_t width, size_t height) {
+  // raster task runner
+  std::unique_lock<std::mutex> lock(resize_mutex_);
+
+  if (resize_status_ != ResizeState::kResizeStarted) {
+    return kWindowFrameBufferID;
+  }
+
+  if (resize_target_width_ == width && resize_target_height_ == height) {
+    surface_manager_->ResizeSurface(GetRenderTarget(), width, height);
+    surface_manager_->MakeCurrent();
+    resize_status_ = ResizeState::kFrameGenerated;
+  }
+
+  return kWindowFrameBufferID;
+}
+
+void FlutterWindowsView::OnWindowSizeChanged(size_t width, size_t height) {
+  // platform task runner
```

Comment:
Does this comment mean this method must be called from the platform thread?  If that is true would be great to flesh out the comment and / or maybe also express this in the header comment for this function since callers (UWP or win32 `FlutterWindow` implementations) have different UI threading models and that contract may have material impact to their implementations.

Reply:
+1; please use style-conformant (i.e., full-sentence) comments. E.g., "Called on the platform thread."

Reply:
Converted these to full comments, and also amended the headers.